### PR TITLE
add support for AZ functions in VS Code

### PR DESCRIPTION
**Pull request by dabutvin**

### CODE REVIEW

**Comment by guntbert** [comment]
Do we **really** want to put `.vscode` into the repo?

**Comment by dabutvin** [comment]
I think we should. It helps to share settings. It's kind of complicated to setup running and debugging the functions without it.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+   // Use IntelliSense to find out which attributes exist for C# debugging
```
Comment (guntbert) [inline .vscode/launch.json]: Why are these lines rendered differently on github? Did you use special characters?

Comment (dabutvin) [reply .vscode/launch.json]: JSON isn't technically allowed to have comments :(

VS Code puts those in there to help though

**Comment by guntbert** [comment]
BTW: thank you for the kind invitation - I'll behave responsibly :hand:

File: .vscode/launch.json (hunk 1 above)
Comment (guntbert) [reply .vscode/launch.json]: makes sense but are you sure that we don't get into trouble parsing the json? I just fund a possible workaround: https://www.quora.com/How-do-I-write-comments-inside-a-JSON-document

Comment (dabutvin) [reply .vscode/launch.json]: haha nice. I can update to do that

**Pull request merged**